### ISSUE TITLE
Feedback: "Bump into the top edge of a modal dialog" (Modal Dialog Example, Test 8, V24.05.09)

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at#1079]
## Description of Behavior

Test fails because vo-home does not move VoiceOver cursor to the top of the dialog. However, if the cursor is move via other means, the test passes. This test needs to be revised to remove the dependency on a cursor movement command that is not essential to the test.

## Test Setup

- Test File: [test-16-bumpTopEdge-voiceover_macos.collected.html](https://aria-at.netlify.app/tests/modal-dialog/test-16-bumpTopEdge-voiceover_macos.collected.html)
- AT: VoiceOver for macOS (Version 14.4.1)
- Browser: Safari (Version 17)

**Comment by mcking65** [comment]
@IsaDC, thank you for #1077, which made changes for "Test 12: Bump into the top edge of a Nested modal dialog". 
After reviewing the new test plan, I recommend making one more change to test 12 and then making the same set of changes to three additional tests.

First, in test 12, change the instructions. They currently read:

>Starting at the 'Address Added' heading inside the nested dialog, navigate to the top of the dialog and make multiple attempts to navigate past the top edge

Please change to:

>Starting at the 'Address Added' heading inside the nested dialog, make multiple attempts to navigate past the top edge

Please also mirror the changes to test 12 in the following tests:

* Test 7: Bump into the bottom edge of a modal dialog
* Test 8: Bump into the top edge of a modal dialog
* Test 11: Bump into the bottom edge of a Nested modal dialog
